cpp/src/io/csv/csv_reader.cu

```diff
@@ -953,11 +958,11 @@ __global__ void storeRecordStart(char *data, const char terminator, const char q
 gdf_error launch_dataConvertColumns(raw_csv_t *raw_csv, void **gdf, gdf_valid_type** valid, gdf_dtype* d_dtypes,string_pair **str_cols, long row_offset, unsigned long long *num_valid) {
 
 	int blockSize;		// suggested thread count to use
-	int minGridSize;	// minimum block count required
-	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, convertCsvToGdf) );
+ 	int minGridSize;	// minimum block count required
+ 	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, convertCsvToGdf) );
 
-	// Calculate actual block count to use based on records count
-	int gridSize = (raw_csv->num_records + blockSize - 1) / blockSize;
+ 	// Calculate actual block count to use based on records count
+ 	int gridSize = (raw_csv->num_records + blockSize - 1) / blockSize;
```

Comment:
Since the previous PR was accepted, merge conflicts resulted in having to manually copy the previous changes back, hence the unintentional extra spaces. Will proceed to fix these trivial things.